medusa-config.ts

```diff
@@ -141,4 +140,7 @@ module.exports = {
   featureFlags: {
     medusa_v2: true,
   },
+  directories: {
+    srcDir: "src",
+  },
```

Comment:
suggestion: it seems to be error prone here if the user end up being able to configure that. do you think it will be removed before we get people to use it and by mistake they miss configure it? like using the env as mentionned in the other pr

Reply:
Yes it will be removed. As mentioned earlier, the goal right now is not break existing apps using the current version of starter kit with the new version of `@medusajs/medusa` package.

> it seems to be error prone here

Curious to know why you think it will be error prone?

Reply:
As I mentioned, I belive we could just compute it on our side in the loader using the env variable, I think it can be error prone cause it is something the user can change and therfore they can put a wrong value or by mistake change it. wdyt

Reply:
So there are two answers.

- Yes, if we can auto detect it, then its great and that's where we may head as well.
- In general, I do not worry too much about someone messing up the config file. Because, the same can happen in other places too. Like, I can mess up Redis URL, or I can mis-type module names.

But yeah, let's remove this config once we are done with the change end-to-end

Reply:
I do agree with you, but I would say that in one case it is user specific and in the other case it is medusa specific, when we can identify something automatically I would prefer to do that instead of asking the user, here it is used to configure how the discovery will work which is something that should be handled under the hood IMO 👍